services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -1078,7 +1102,7 @@ public void sendPushJobDetails(PushJobStatusRecordKey key, PushJobDetails value)
           VeniceSystemStoreUtils.getPushJobDetailsStoreName(),
           value.getSchema().toString());
       return getVeniceWriterFactory().createVeniceWriter(
-          new VeniceWriterOptions.Builder(pushJobDetailsRTTopic)
+          new VeniceWriterOptions.Builder(pushJobDetailsRTTopic.getName())
```

Comment:
I think we should just update builder to use `PubSubTopic` object instead of topic name string